exchanges/order/order_test.go

```diff
@@ -400,71 +351,42 @@ func TestInferCostsAndTimes(t *testing.T) {
 	detail.Amount = 1
 	detail.RemainingAmount = 1
 	detail.InferCostsAndTimes()
-	if detail.Amount != detail.ExecutedAmount+detail.RemainingAmount {
-		t.Errorf(
-			"Order detail amounts not equals. Expected 0, received %f",
-			detail.Amount-(detail.ExecutedAmount+detail.RemainingAmount),
-		)
-	}
+	assert.Equal(t, detail.ExecutedAmount+detail.RemainingAmount, detail.Amount)
 	detail.RemainingAmount = 0
 
 	detail.Amount = 1
 	detail.ExecutedAmount = 1
 	detail.Price = 2
 	detail.InferCostsAndTimes()
-	if detail.AverageExecutedPrice != 2 {
-		t.Errorf(
-			"Unexpected AverageExecutedPrice. Expected 2, received %f",
-			detail.AverageExecutedPrice,
-		)
-	}
+	assert.Equal(t, 2., detail.AverageExecutedPrice)
```

Comment:
```suggestion
	assert.Equal(t, 2.0, detail.AverageExecutedPrice)
```

With decimal equality I think it's clearer if you do it like this and for other tests